extensions/amp-copy/1.0/component.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '#preact';
+import {
+  useCallback,
+  useMemo,
+  useState,
+} from '#preact';
+import {
+  copyTextToClipboard,
+  isCopyingToClipboardSupported,
+} from '../../../src/clipboard';
+import {useStyles} from './component.jss';
+
+/**
+ * @param {!CopyDef.Props} props
+ * @return {PreactDef.Renderable}
+ */
+export function Copy({children, sourceId, text, ...rest}) {
```

Comment:
What is the purpose of the `text` prop?

Reply:
We can pass static text to the `text` attribute, which will then get copied when you click the button.
This will work only when the `sourceId` is missing or not passed.

Reply:
Thank you!